public/app/plugins/panel/table/module.tsx

```diff
@@ -59,24 +62,22 @@ export const plugin = new PanelPlugin<PanelOptions, TableFieldOptions>(TablePane
           },
           defaultValue: defaultPanelFieldConfig.align,
         })
-        .addSelect({
-          path: 'displayMode',
-          name: 'Cell display mode',
-          description: 'Color text, background, show as gauge, etc',
-          settings: {
-            options: [
-              { value: TableCellDisplayMode.Auto, label: 'Auto' },
-              { value: TableCellDisplayMode.ColorText, label: 'Color text' },
-              { value: TableCellDisplayMode.ColorBackground, label: 'Color background (gradient)' },
-              { value: TableCellDisplayMode.ColorBackgroundSolid, label: 'Color background (solid)' },
-              { value: TableCellDisplayMode.GradientGauge, label: 'Gradient gauge' },
-              { value: TableCellDisplayMode.LcdGauge, label: 'LCD gauge' },
-              { value: TableCellDisplayMode.BasicGauge, label: 'Basic gauge' },
-              { value: TableCellDisplayMode.JSONView, label: 'JSON View' },
-              { value: TableCellDisplayMode.Image, label: 'Image' },
-            ],
+        .addCustomEditor<void, TableCellOptions>({
+          id: 'cellOptions',
+          path: 'cellOptions',
+          name: 'Cell Options',
```

Comment:
I think this will need a different name because it's the same as the cell category and can cause confusion. E.g in overrides:
![Screenshot 2022-12-20 at 08 55 04](https://user-images.githubusercontent.com/36818606/208616003-23444a27-3eea-47e7-8e7c-fe4c5652b221.png)

Reply:
This indeed confusing, I updated the editor to be named "Cell Type" though I had to sacrifice the description on the form field as the `Field` component will only show the description if it has a name 😢